pom.xml

```diff
@@ -579,6 +565,36 @@
     </distributionManagement>
 
     <profiles>
+        <profile>
+            <id>api</id>
+            <activation>
+                <property>
+                    <name>!skipApi</name>
+                </property>
+            </activation>
+            <modules>
+                <module>kroxylicious-bom</module>
```

Comment:
Does this actually work? The bom is going to contain references to non API modules as snapshots as well so can't be released. 

I'm starting to think we can't dog food the bom within the main project tree it has to be a independent artefact.